.coderabbit.yaml

```
@@ -0,0 +1,80 @@
+
+# Comment settings
+comments:
+  # Enable or disable inline comments
+  inline: true
+  # Enable or disable summary comments
+  summary: true
+  # Customize comment templates
+  templates:
+    summary: |
+      ## CodeRabbit Review Summary
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)